packages/babel-compat-data/scripts/build-modules-support.js

```diff
@@ -41,7 +41,11 @@ function process(source) {
     if (Array.isArray(browserSupport)) {
       browserSupport = browserSupport[0]; // The first item is the most progressive support
     }
-    if (browserSupport.version_added && !browserSupport.flags) {
+    if (
+      browserSupport.version_added &&
+      !browserSupport.flags &&
+      !browserSupport.partial_implementation
```

Comment:
It is used to rule out Safari 14 on class properties transform. It is an no-op in this PR.

https://github.com/mdn/browser-compat-data/blob/178137547bc29a79b712cec221af099329b1f4a0/javascript/classes.json#L590